### PR TITLE
Update readme and contributing guideline.

### PR DESCRIPTION
**Pull request by st0012**
- I move the bytecode specifications into [wiki](https://github.com/goby-lang/goby/wiki/Bytecode-Instruction-specs).
- I also removed sample code snippets from readme cause I think there are very few people will try them.
- Updated some outdated infos.

Please help me check the grammar and link @adlerhsieh @janczer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,13 +100,6 @@ Although we definitely will replace current parser with a parser generator, this
...
-
-## TODO & WIP
-
-Checkout this [issue](https://github.com/goby-lang/goby/issues/72) for what we will work on before first release.
-
-Also see [huboard](https://huboard.com/goby-lang/goby)
-
 ## Install
 
 1. You must have Golang installed
```
Comment (janczer) [inline README.md]: Maybe should change `Golang` to `[Go](https://golang.org/)`?

Comment (st0012) [inline README.md]: I think people can understand what I'm talking about if they're already reading this line 😅. So I don't think we need to change that.

Comment (janczer) [inline README.md]: Ok :)

**Comment by janczer** [comment]
LGTM

**Pull request merged**